praetorian_cli/ui/aegis/commands/job.py

```
@@ -226,6 +243,226 @@ def run_job(menu, args):
+        menu.console.print(f"  Target:      {dns}")
+        menu.console.print(f"  Job Key:     {job_key}")
+        menu.console.print(f"  Created:     {format_timestamp(created, '%Y-%m-%d %H:%M:%S')}")
+        if updated:
+            menu.console.print(f"  Updated:     {format_timestamp(updated, '%Y-%m-%d %H:%M:%S')}")
+
+        # Show config if present (minus sensitive fields)
+        if config and isinstance(config, dict):
+            display_config = {k: v for k, v in config.items()
+                              if k.lower() not in ('password', 'secret', 'token')}
+            if display_config:
+                menu.console.print(f"\n  Configuration:")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)